src/main/perl/Proc.pm

```diff
@@ -0,0 +1,10 @@
+#${PMpre} NCM::Proc${PMpost}
+
+use parent qw(NCM::DummyBlockdevice);
+
+sub devpath
+{
+    return "tmpfs";
```

Comment:
Shouldn't this be "proc" not tmpfs?

Reply:
@stdweird?

Reply:
EL6 - `proc on /proc type proc (rw)`
EL7 - `proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)`
EL8 - `proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)`